scripts/webpack.config.js

```diff
@@ -45,3 +45,41 @@ module.exports = {
   devtool: 'cheap-module-source-map',
   target: 'web'
 };
+
+const expoConfig = {
+  entry: {
+    otplib: directory.SOURCE + '/otplib-expo/index.js'
+  },
+  output: {
+    library: '[name]',
+    libraryTarget: 'umd',
+    path: directory.BUILD,
+    filename: 'otplib-expo.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js?$/,
+        exclude: /node_modules/,
+        use: ['babel-loader']
+      }
+    ]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(ENV)
+    }),
+    new webpack.optimize.UglifyJsPlugin({
+      compress: {
+        warnings: false
+      }
+    }),
+    new webpack.BannerPlugin({
+      banner: createBanner('otplib-expo'),
+      raw: true
+    })
+  ],
+  devtool: 'cheap-module-source-map'
+};
+
+module.exports = [webConfig, expoConfig];
```

Comment:
since both configurations are the same, and i foresee that any other packages should also follow the same pattern, it would be better to have it as a function creator.

i.e. 
```js
function createConfig(name) {
  return {
    .....
  }
}

module.exports = [
  createConfig('otplib-browser'),
  createConfig('otplib-expo')
];
```

Reply:
I agree. I was playing with the idea of just merging the common parts with the specific ones using Object.assign but function works is better suited as the string has to be added several times.